spring-cloud-starter-dataflow-server/pom.xml

```diff
@@ -82,6 +82,14 @@
 			</resource>
 		</resources>
 		<plugins>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<excludes>
+						<exclude>**/local/security/**.java</exclude>
```

Comment:
The classes in local.security package all had `@Ignore` and it seems as if JUnit 5 will initialize the class and only then decide to ignore the tests. It caused failures because they couldn't find an LDAP server.